edx_django_utils/hooks/triggers.py

```diff
@@ -0,0 +1,45 @@
+"""
+TODO: add description
+"""
+from logging import getLogger
+
+from .pipeline import run_pipeline
+from .utils import get_pipeline_configuration
+
+
+def trigger_filter(trigger_name, *args, **kwargs):
```

Comment:
What do you think about this:

We had before:

https://github.com/eduNEXT/edx-django-utils/blob/e192affb6a3b582edad6b3d324df4bc26492cd44/edx_django_utils/hooks/pipeline.py#L12
https://github.com/eduNEXT/edx-django-utils/blob/e192affb6a3b582edad6b3d324df4bc26492cd44/edx_django_utils/hooks/pipeline.py#L32
https://github.com/eduNEXT/edx-django-utils/blob/e192affb6a3b582edad6b3d324df4bc26492cd44/edx_django_utils/hooks/pipeline.py#L50

and

https://github.com/eduNEXT/edx-django-utils/blob/e192affb6a3b582edad6b3d324df4bc26492cd44/edx_django_utils/hooks/triggers.py#L32
https://github.com/eduNEXT/edx-django-utils/blob/e192affb6a3b582edad6b3d324df4bc26492cd44/edx_django_utils/hooks/triggers.py#L12

In `run_filter_pipeline` and `run_action_pipeline` are not defining a pipeline, we are just changing how we call it depending on the trigger, that's why I changed those to just trigger_action and trigger_filter and made run_pipeline _public_

Reply:
I think that it is fine!

At the end of the day we will need to create a tasks.py file and move the run_pipeline to that place.